PangleAdapter/src/main/java/com/chartboost/mediation/pangleadapter/PangleAdapter.kt

```diff
@@ -9,17 +9,68 @@ package com.chartboost.mediation.pangleadapter
 
 import android.app.Activity
 import android.content.Context
-import android.view.View
-import com.bytedance.sdk.openadsdk.*
-import com.chartboost.heliumsdk.domain.*
+import android.util.Size
+import com.bytedance.sdk.openadsdk.api.PAGConstant
+import com.bytedance.sdk.openadsdk.api.banner.PAGBannerAd
```

Comment:
@CB-RyanMcCormick do you refer to these?? I don't know why, but my IDE is starting to do this now. 😱 Does it happen to any of you?? 👀